src/rgw/rgw_process_env.h

```diff
@@ -36,8 +39,13 @@ struct RGWLuaProcessEnv {
   std::unique_ptr<rgw::sal::LuaManager> manager;
 };
 
+struct RGWDedupEnv {
```

Comment:
since we are not passing the dedup background pointer anywhere, we probably don't need the `RGWDedupEnv` class